dts/msm8916/msm8916-asus-z010d.dts

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "skeleton.dtsi"
+
+/ {
+	qcom,msm-id = <206 0>;
+	qcom,board-id = <0x8 0x0>;
```

Comment:
This is the same as `msm8916-mtp.dts` so you need to add an entry there and match something from the command line. You can check the command line using `fastboot oem cmdline && fastboot get_staged /dev/stdout`.

Reply:
This the the output of ```fastboot oem cmdline && fastboot get_staged /dev/stdout```

```
lk2nd console=logk0 androidboot.cpu.serialno=14e4d011  androidboot.asus.warranty=0  CPU_RV=007050e1 CPU_FEATURE_ID=11 loglevel=0 androidboot.cpu.security=enable  pcb_rev=0  androidboot.emmc.type=16G  SB=Y androidboot.rf.sku.id=2 androidboot.serialno=G7AXB7C162483SF androidboot.emmc=true androidboot.mode=charger_shipping androidboot.baseband=msm mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_r69339_720p_video:1:noneOKAY
```

I'm guessing CPU_FEATURE_ID=11 is the one I'm looking for

Reply:
That one might work, alternatively you can also match `mdss_mdp.panel=` if you want.

Reply:
Done.

Reply:
you still need to move it to `msm8916-mtp.dts`

Reply:
I think this should work

Reply:
Should I squash these commits into one?

Reply:
> I think this should work

Please test that it works :)

> Should I squash these commits into one?

Yes, please :)

Reply:
> Please test that it works :)

Did that already :+1: 

> Yes, please  :)

In a minute